itk_napari_conversion.py

```diff
@@ -62,16 +62,34 @@ def image_from_image_layer(image_layer):
     return image
 
 
-def points_layer_from_point_set(point_set):
-    """Convert an itk.PointSet to a napari.layers.Points."""
+def points_layer_from_point_set(point_set, reverse_coords=True):
+    """Convert an itk.PointSet to a napari.layers.Points.
+
+    Parameters
+    ----------
+    point_set : itk.PointSet
+        The ITK PointSet to convert.
+    reverse_coords : bool, optional
+        If True (default), reverse the coordinate order from ITK's x,y,z
+        to napari's z,y,x order.
+
+    Returns
+    -------
+    napari.layers.Points
+        The converted napari Points layer.
+    """
     # Get points as numpy array
     number_of_points = point_set.GetNumberOfPoints()
 
     if number_of_points == 0:
         data = np.array([]).reshape(0, 3)  # Default to 3D empty array
     else:
         points_array = itk.array_from_vector_container(point_set.GetPoints())
-        data = points_array
+        if reverse_coords:
+            # Reverse coordinate order from ITK (x,y,z) to napari (z,y,x)
+            data = points_array[:, ::-1]
+        else:
+            data = points_array
```

Comment:
This is the main part of the fix, right? Cool! I can't wait to try it out, with https://github.com/SuperElastix/elastix-napari/pull/65 😃